flytekit/clis/sdk_in_container/init.py

```diff
@@ -0,0 +1,38 @@
+import click
+from cookiecutter.main import cookiecutter
+from cookiecutter.prompt import read_user_variable
+
+
+@click.command("init")
+@click.option("--template", default="simple-example", help="cookiecutter template to be used")
+@click.argument("project-name")
+def init(template, project_name):
+    """
+    Create flyte-ready projects.
+    """
+    click.echo(
+        "What should we call your application? This serves as the top level package where your workflows will live."
+    )
+    app = read_user_variable("app", "myapp")
+    click.echo("What should be the name of your example workflow?")
+    workflow_name = read_user_variable("workflow", "workflow_example")
```

Comment:
Should we just use click for consistency http://pallets-click.readthedocs.io/en/stable/prompts.html

Reply:
using cookiecutter just for the templating feels less cluttered. Thanks for the suggestion!